proposed/http-factory/http-factory.md

```diff
@@ -163,14 +170,17 @@ fwrite($resource, $body);
 Has the ability to create streams for uploaded files.
 
 ```php
-namespace Psr\Http\Message;
+namespace Psr\Http\Factory;
+
+use Psr\Http\Message\UploadedFileInterface;
 
 interface UploadedFileFactoryInterface
 {
     /**
      * Create a new uploaded file.
      *
-     * If a string is passed it is assumed to be a file path.
+     * If a string is used to create the file, a temporary resource will be
+     * created with the content of the string.
```

Comment:
I have not enough knowledge about this PSR, but I'm wondering if a `StreamInterface` method don't/shouldn't need the `$size` parameter? – If that is the case, then I believe, two separate methods make sense.